extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.tsx

```diff
@@ -149,13 +149,7 @@ function TrackedMeasurementsContextProvider(
       }
     },
     clearAllMeasurements: (ctx, evt) => {
-      const measurements = measurementService.getMeasurements();
-      const measurementIds = measurements.map(fm => fm.uid);
-
-      for (let i = 0; i < measurementIds.length; i++) {
-        measurementService.remove(measurementIds[i]);
-      }
-      measurementService.setIsMeasurementDeletedIndividually(false);
```

Comment:
why are you removing this setIsMeasurementDeletedIndividually?

Reply:
I didn't remove this line specifically. I replaced the whole code here that should be clearing all measurements with the `clearAllMeasurements` method that already exists and that I think serves the same purpose.

However, I see now that you commented here that this particular line is not present in the `clearAllMeasurements` method. I can include it back. Do you want it added back here or inside the `clearAllMeasurements` method?